### ISSUE TITLE
nn.Dropout hidden by rnn.Dropout

### ISSUE THREAD

**Issue by saiganeshb** [Element-Research/rnn#296]
Hi,
I'm using the rnn library for LSTM, but want to use the regular nn.Dropout module for the rest of my net. It's currently being hidden by the rnn module's Dropout implementation (https://github.com/Element-Research/rnn/blob/master/Dropout.lua). Is there a short term workaround for this?

 Also for the long-term, I think it would be better to rename the modules so there are no conflicts and it's clear for new users where these modules are getting picked up from.
